composer.json

```diff
@@ -10,14 +10,16 @@
         }
     ],
     "require": {
+        "php": "^7.2",
+        "laravel/framework": "^5.7|^6.0",
         "moneyphp/money": "^3.0",
         "ramsey/uuid": "^3.7"
     },
     "require-dev": {
-        "mockery/mockery": "^0.9.5",
-        "orchestra/testbench": "~3.4.0",
-        "orchestra/database": "~3.4.0",
-        "phpunit/phpunit": "^5.2"
+        "mockery/mockery": "^1.0",
+        "orchestra/testbench": "~4.0.0",
+        "orchestra/database": "~4.0.0",
```

Comment:
v5.0.0 exists?
https://packagist.org/packages/orchestra/database

why?

Reply:
because it requires laravel 7 version.